contracts/on-chain-claim/src/config.rs

```diff
@@ -24,13 +25,43 @@ pub trait ConfigModule {
 
     #[view(getAddressInfo)]
```

Comment:
Using the new vector mapper, I'd delete this and just add a view annotation on top of the storage, to just have a way to manually check the date of a user. Or you could do this later on, if we manage to migrate all users.